pyhoma/client.py

```diff
@@ -77,10 +82,9 @@ async def login(self) -> bool:
         response = await self.__post("login", data=payload)
 
         if response.get("success"):
-            self.__roles = response.get("roles")
-
+            if register:
```

Comment:
Can we change `register` to something which is more descriptive? `register_event_listener` or something similar?

Reply:
I hesitate, I don't like long name. But you are right.